machine.go

```diff
@@ -880,3 +869,38 @@ func (m *Machine) waitForSocket(timeout time.Duration, exitchan chan error) erro
 		}
 	}
 }
+
+// Set up a signal handler to pass through to firecracker
+func (m *Machine) setupSignals() {
+	signals := m.Cfg.ForwardSignals
+
+	if signals == nil {
```

Comment:
Can we move this to `NewMachine`? That's where all defaults are occurring and I'd like to keep them in the same area.

Reply:
Good call, fixed.